### PR TITLE
Deleting multiple layers issue #317

### PR DESCRIPTION
**Pull request by Gurpreetsingh9465**
This is a PR regarding issue #317 

1. multiple layer selection
2. deletion
3. confirmation
4. improved the logic of tracking nextlayer
5. making sure the the default select layer method does not conflict with this

### CODE REVIEW

**Comment by yashdusing** [comment]
How do you select the layers ?

**Comment by Gurpreetsingh9465** [comment]
Right click selects the layer

**Comment by Gurpreetsingh9465** [comment]
Right now the code is in a way that If sidepane is enabled then you can't select or deselect layers further and the the sidepanel layer is deselected.

**Comment by Gurpreetsingh9465** [comment]
The new layer is appending I guess you can see it on the .gif as well can you send me the console output

**Comment by yashdusing** [comment]
Sorry, I was a little busy. Here you go. 
screenshot_20181208_004938

**Comment by yashdusing** [comment]
Steps to recreate :
1. Load ZFnet
2. Select random layers from the middle and then delete these layers
3. Then select one of the 2 remaining sets and delete it.
4. Now try adding layers

**Comment by yashdusing** [comment]
Also, please do something about the css. It's hard to keep track of the selected layers

**Comment by Gurpreetsingh9465** [comment]
test
updated the css

**Comment by Gurpreetsingh9465** [comment]
I noticed that there is a problem in tracking the last the layers of the models which are loaded from the model zoo . this things even happens for the single layer deletion (old code) for this i am opening a new issue as this issue is nothing to do with the delete layer code.

File: README.md [hunk 1]
```diff
@@ -299,6 +299,7 @@ Models                                                                      | Ca
 [Pix2Pix](https://github.com/phillipi/pix2pix)                              |   √   |   ×   |     ×      |
 [VQA](https://github.com/iamaaditya/VQA_Demo)                               |   √   |   √   |     √      |
 [Denoising Auto-Encoder](https://blog.keras.io/building-autoencoders-in-keras.html)                               |   ×   |   √   |     √      |
+[Vnect](https://arxiv.org/abs/1705.01583)                               |   √   |   ×   |     ×      |
```
Comment (Ram81) [inline README.md]: Please remove the changes for adding a new model

File: ide/static/js/canvas.js [hunk 2]
```diff
@@ -380,6 +384,31 @@ class Canvas extends React.Component {
     }
     this.props.setDraggingLayer(null);
   }
+  addLayerToSelectedList(id) {
+    if(this.props.selectedLayer != id) {
+      this.props.selectedLayers.push(id);
+      this.setState({
+        selectCount:this.state.selectCount+1
```
Comment (Ram81) [inline ide/static/js/canvas.js]: Add space after `:`, see the style changes from other files

**Comment by Gurpreetsingh9465** [comment]
Changes done.

**Pull request closed without merging**